docs/source/55_api.md

```diff
@@ -339,6 +339,52 @@ val bcastFeatures = sc.loadFeatures("my/features.adam").broadcast()
 val readsByFeature = reads.broadcastRegionJoinAgainst(bcastFeatures)
 ```
 
+#### Examples of real-world analyses possible with the RegionJoin API
+###### Filter Genotypes by Features
+
+```scala
+// Inner join will filter out genotypic data not represented in the feature dataset
+val genotypes = sc.loadGenotypes(“my/genotypes.adam”)
+val features = sc.loadFeatures(“my/features.adam”)
+
+// We can use ShuffleRegionJoin…
+val filteredGenotypes_shuffle = genotypes.shuffleRegionJoin(features)
```

Comment:
Can you `s/_shuffle/Shuffle/g` throughout this PR? Ditto with `s/_bcast/Bcast/g`.